docs/apache-airflow/kubernetes.rst

```diff
@@ -20,31 +20,62 @@
 Kubernetes
 ----------
 
+Apache Airflow aims to be a very Kubernetes-friendly project, and many users run Airflow
+from within a Kubernetes cluster in order to take advantage of the increased stability
+and autoscaling options that Kubernetes provides.
+
+
+Helm Chart
+^^^^^^^^^^
+
+`Helm https://helm.sh/`_ provides a simple mechanism to deploy software to a
+Kubernetes cluster, and the community is working on an
+`offical Helm chart https://github.com/apache/airflow/tree/master/chart`_ for Airflow
+which will be released with Airflow 2.1. The chart has
+`KEDA integration https://github.com/apache/airflow/tree/master/chart#autoscaling-with-keda`_
+which allows you to autoscale your Celery workers. More information is available
+`in this blog post https://www.astronomer.io/blog/the-keda-autoscaler`_.
+
+Other Helm charts in the wild:
+
+* https://github.com/airflow-helm/charts
+* https://bitnami.com/stack/apache-airflow/helm
```

Comment:
I don't know if we want links to third-party products to be included in the documentation. I don't know if Apache or this project has a rule on this, but the CNCF doesn't allow such references.
https://kubernetes.io/docs/contribute/style/content-guide/
https://github.com/kubernetes/website/issues/15576
https://github.com/kubernetes/website/issues/15821

Reply:
We have one page on the site that allows you to post this type of content.
https://airflow.apache.org/ecosystem/